CONTRIBUTING.md

```diff
@@ -1,18 +1,83 @@
+These components are part of the JSCAD Organization, and is maintained by a group of volunteers. We welcome and encourage anyone to pitch in but please take a moment to read the following guidelines.
 
-* We only accept bug reports and pull requests on GitHub*.
+* If you want to submit a bug report please make sure to follow the [Reporting Issues](https://github.com/jscad/csg.js/wiki/Reporting-Issues) guide. Bug reports are accepted as [Issues](https://github.com/jscad/OpenJSCAD.org/issues/) via GitHub.
 
-* If you want to contribute to OpenJSCAD.org documentation, you can start immediately as new contributions are accepted on Wikibooks via the online [OpenJSCAD User Guide](https://en.wikibooks.org/wiki/OpenJSCAD_User_Guide).
+* If you want to submit a change or a patch, please read the contents below on how to make changes. New contributions are accepted as [Pull Requests](https://github.com/jscad/OpenJSCAD.org/pulls/) via GithHub.
 
-* If you have a question about how to use OpenJSCAD.org, please start a conversation at the [OpenJSCAD.org User Group](https://plus.google.com/communities/114958480887231067224). Please read the [OpenJSCAD.org User Guide](https://github.com/Spiritdude/OpenJSCAD.org/wiki/User-Guide) beforehand.
+* We only accept bug reports and pull requests on **GitHub**.
 
-* If you have a change or new feature in mind, please start a converstation with the [Core Developers](https://plus.google.com/communities/114958480887231067224) and start contributing changes.
+* If you have a question about how to use OpenJSCAD, then please start a conversation at the [OpenJSCAD.org User Group](https://plus.google.com/communities/114958480887231067224). You might find the answer in the [OpenJSCAD.org User Guide](https://github.com/Spiritdude/OpenJSCAD.org/wiki/User-Guide).
 
-Thanks! :heart: :heart: :heart:
+* If you have a change or new feature in mind, please start a conversation with the [Core Developers](https://plus.google.com/communities/114958480887231067224) and start contributing changes.
 
-The OpenJSCAD.org Core Developers
+Thanks!
+
+The JSCAD Organization
+
+## Making Changes
+
+First, we suggest that you fork this GIT repository. This will keep your changes separate from the fast lane.  And make pull requests easier.
+
+Once forked, clone your copy of the OpenJSCAD.org components.
+```
+git clone https://github.com/myusername/OpenJSCAD.org.git
+cd OpenJSCAD.org
+```
+
+**We suggest downloading NPM. This will allow you to generate API documents, and run test suites.**
+
+Once you have NPM, install all the dependencies for development.
+```
+npm install
+```
+And, run the test cases to verify the library is functional.
+```
+npm test
+```
+Next, start a small webserver to run the website. (Note: The web pages will automatically reload if changes are made.)
+```
+npm run start-dev
+```
+
+The project is structured as:
```

Comment:
This makes me think we should perhaps have folders per build type (standard, min, options) so that they are all together and user just need to copy and paste the folder